frontend/src/store/Domains/actions.js

```diff
@@ -19,7 +19,7 @@ export const addDomain = async ({ commit }, domain) => {
 	commit("NEW_DOMAIN", response.data);
 };
 export const updateDomain = async ({ commit }, domain) => {
-	let response = await API.put(`/domain/${domain.id}`, domain);
```

Comment:
Export the rename of  `id` to `ID` in another commit.

Reply:
ok